apsuite/commisslib/control_rf.py

```diff
@@ -11,14 +11,16 @@
 import pyaccel as _pyaccel
 from pymodels import si as _si, bo as _bo
 
-from ..utils import ThreadedMeasBaseClass as _BaseClass
+from ..utils import ThreadedMeasBaseClass as _BaseClass, ParamsBaseClass as \
```

Comment:
@murilobalves, could you break the line before `ParamsBaseClass` ?

Reply:
done